TestAssets/TestProjects/PortableApp/project.json

```diff
@@ -9,7 +9,7 @@
       "dependencies": {
         "Microsoft.NETCore.App": {
           "type": "platform",
-          "version": "1.0.0"
+          "version": "1.0.0-*"
```

Comment:
@eerhardt is right, but you also need the `1.0.0-servicing-*` to end up in the `project.lock.json`. But because you are disabling the building of those packages by setting the `BuildAllPackages` to `false` the NuGet/MyGet feed wins instead of the local just built package.

I would suggest that we produce all the packages, but prevent publishing them, rather than stop producing them. Or it might be better to create a staging feed/blob storage for testing purposes and then move them to the main feeds.